opal/templates/patient_lists/tabbed_list_group_base.html

```diff
@@ -0,0 +1,11 @@
+<div class="patient-list-group">
+  <ul class="nav nav-tabs nav-justified" role="tablist">
+    {% for member in members %}
+    <li role="presentation" {% if member.display_name == active_list.display_name %}class="active"{% endif %}>
+      <a href="#/list/{{ member.get_slug }}" aria-controls="home" role="tab" data-toggle="tab">
```

Comment:
I have literally no idea - I cargo culted it from the example markup here here: http://getbootstrap.com/javascript/#tabs